capstone/src/main/webapp/script.js

```diff
@@ -0,0 +1,57 @@
+function getDisplay() {
+  fetch("/display").then(response => response.json()).then((list) => {
+    // list is an arraylist containing strings, so we have to
+    // reference its elements to create HTML content
+    const contentListElement = document.getElementById("displayComponent");
+    contentListElement.innerHTML = '';
+
+    for (var i = 0; i < list.length; i++) {
+      contentListElement.appendChild(
+      createListElement(('Res: '+ list[i].resFileSize +'bytes')));
+      contentListElement.appendChild(
+      createListElement(('Dex: '+ list[i].dexFileSize +'bytes')));
+      contentListElement.appendChild(
+      createListElement(('Lib: '+ list[i].libraryFileSize +'bytes')));
+      contentListElement.appendChild(
+      createListElement(('Ass: '+ list[i].assetsFileSize +'bytes')));
```

Comment:
Let's call these something clearer. Like "Assets", "Java Code" (for DEX), etc...

Reply:
Done. Thanks